css/types/attr.json

```
@@ -186,9 +193,125 @@
               }
             }
           },
+          "custom-ident": {
+            "__compat": {
+              "description": "`<custom-ident>`",
+              "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/Reference/Values/custom-ident",
+              "spec_url": "https://drafts.csswg.org/css-values/#custom-idents",
+              "tags": [
+                "web-features:attr"
+              ],
+              "support": {
+                "chrome": {
+                  "version_added": false
```

Supported in Chrome 133 (see [here](https://github.com/chromium/chromium/commit/01b72da0c0fa34489790d3e479ee89c5c09fb8a9#diff-067cc67f1c5bc4790769a2d7348c292358a2fc05746d2ecf5547a058f5586692R96)):

```suggestion
                  "version_added": "133"
```